src/lib.rs

```diff
@@ -289,9 +289,17 @@ impl SiPrefix {
 #[allow(missing_docs)]
 #[derive(Eq, PartialEq, Debug, Clone)]
 pub enum Currency {
+	/// Bitcoin mainnet
 	Bitcoin,
+
+	/// Bitcoin testnet
 	BitcoinTestnet,
+
+	/// Bitcoin regtest
 	Regtest,
+
+	/// Bitcoin (`btcd`) simnet
```

Comment:
It's not a `btcd`-only thing afaik. At least the [`signet` feature](https://en.bitcoin.it/wiki/Signet) in core seems to use the same prefix (or at least the [people using it](https://explorer.bc-2.jp/address/sb1qr7pvknzy3netgmv0gpdaptxcev648x4wmatnyj)).

Reply:
Sure, thanks for the clarification!